google/cloud/bigtable/client.py

```diff
@@ -166,16 +162,6 @@ def __init__(
         self._admin = bool(admin)
         self._client_info = client_info
         self._emulator_host = os.getenv(BIGTABLE_EMULATOR)
-        self._emulator_channel = None
-
-        if self._emulator_host is not None:
-            self._emulator_channel = grpc.insecure_channel(
```

Comment:
the change moves most of this to a method. Also, using the method derived in firestore, a secure channel can be used.